joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/MiscTests.scala

```diff
@@ -269,4 +269,26 @@ class MiscTests extends RubyCode2CpgFixture {
       cpg.typeDecl.name("some_class").size shouldBe 1
     }
   }
+
+  "CPG for code with method ending with =" should {
+    val cpg = code("""
+        |class MyClass
+        | def foo=(value)
+        | puts value
+        | end
+        |end
+        |
+        |obj = MyClass.new
+        |obj.foo="arg"
+        |""".stripMargin)
+
+    "recognise all call nodes" in {
+      cpg.call.name("puts").size shouldBe 1
+      cpg.call.name("<operator>.fieldAccess").size shouldBe 1
```

Comment:
I'm ok accepting right now, but the idea behind a method whose name ends in `=` is that `x.foo=arg` becomes `x.foo(arg)`, so the string `"arg"` should taint `puts`.

Please document this so that we know where to look for later on.